ddl/backfilling.go

```diff
@@ -439,6 +439,9 @@ func (dc *ddlCtx) sendTasksAndWait(scheduler *backfillScheduler, totalAddedCount
 	batchTasks []*reorgBackfillTask) error {
 	reorgInfo := scheduler.reorgInfo
 	for _, task := range batchTasks {
+		if scheduler.copReqSenderPool != nil {
+			scheduler.copReqSenderPool.sendTask(task)
+		}
 		scheduler.taskCh <- task
```

Comment:
Should the task be sent to two different channels?

Reply:
I think so. This is because I want to keep the old backfill implementation and introduce as few changes as possible:

Old backfill:  
```
       scheduler.taskCh -> backfill worker -> read rows -> write indexes -> resultCh
```
New backfill in this PR:  
```
       scheduler.taskCh -> backfill worker -> write indexes -> resultCh
                                             ^
                                             |
copReqSenderPool.taskCh -> cop-req sender -> read rows
```

If we only use one channel, then it will be

```
       scheduler.taskCh -> backfill worker -> write indexes -> resultCh
                        |                    ^
                        v                    |
                        -> cop-req sender -> read rows
```

The backfill worker cannot process the next task until the current task finished. However, this is not the same behavior we want in the cop-request sender pool. To gain better performance, the sender pool should be always filled with tasks.

Besides, one channel is not easy to set the concurrency of backfill worker and cop-request sender separately.